corpus_descriptions_test/portage_masked.list

```diff
@@ -0,0 +1,15 @@
+app-office_skrooge
```

Comment:
What is this file for?

Reply:
`mask` is a flag in Portage, which means this packages is "masked" so that you cannot build/install it.
Will remove it with the pkg.list since we have corresponding scripts.